src/renderer/base/lib/base.vcxproj

```diff
@@ -6,7 +6,7 @@
     <RootNamespace>base</RootNamespace>
     <ProjectName>RendererBase</ProjectName>
     <TargetName>ConRenderBase</TargetName>
-    <ConfigurationType>StaticLibrary</ConfigurationType>
+    <ConfigurationType>StaticLibrary</ConfigurationType>

```

Comment:
If you’re wondering why this is modified, originally I added an “interface” class here but then found it useless. VS modified the line ending anyway

Reply:
Since this is the only change in the file, would you mind reverting it?

Reply:
Sure. Will do

Reply:
It took a while for me to realize I'm actually changing the LF here to CRLF, to make it 'correct'. Do you still want me to revert this?...

Reply:
Oh. Since this makes it correct, you can keep it.